kernel-default.properties

```diff
@@ -205,8 +200,8 @@ keymanager_database_url = jdbc:postgresql://${mosip.kernel.database.hostname}:${
 keymanager_database_password=${db.dbuser.password}
 keymanager_database_username= keymgruser
 
-otpmanager_database_username = kerneluser
-otpmanager_database_url = jdbc:postgresql://${mosip.kernel.database.hostname}:${mosip.kernel.database.port}/mosip_kernel
+otpmanager_database_username=otpuser
+otpmanager_database_url=jdbc:postgresql://${mosip.kernel.database.hostname}:${mosip.kernel.database.port}/mosip_otp
```

Comment:
do we have a migration script for this db movement ?

Reply:
Yes... it's here. https://github.com/mosip/otp-manager/blob/release-1.3.x/db_upgrade_scripts/mosip_otp/sql/upgrade.sh